talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -312,6 +312,21 @@ bool P2PClient::IsPeerConnectionChannelCreated(const std::string& target_id) {
     return false;
   return true;
 }
+bool P2PClient::IsPeerConnectionStale(const std::string& target_id) {
+  std::unique_lock<std::mutex> lock(pc_channels_mutex_);
+
+  // Try to find peer conntction
+  auto pcc_it = pc_channels_.find(target_id);
+
+  // If can't find it, return true so AmbientClient cleans up its map
```

Comment:
Comment shouldn't be specific to our implementations, should refer to "Signaling Channel" not "AmbientClient"

Reply:
done